pages/index.vue

```diff
@@ -13,23 +13,33 @@
     />
     <section class="container">
       <div class="mt-3 w-100">
-        <ContentCardSection
-          v-for="section in page.hasPart"
-          :key="section.sys.id"
-          :section="section"
-        />
+        <template v-if="isRichText">
+          <RichText
+            :headline="page.hasPart[0].fields.headline"
+            :text="page.hasPart[0].fields.text"
+          />
+        </template>
+        <template v-else>
```

Comment:
We should stop assuming that sections will be of a particular type, by here using `v-if-else` (type is cardGroup), and having no `v-else`. The content would not be rendered for any other sections, but the rest of the page would not fail to load.

Reply:
See code sample at https://github.com/europeana/incubator-portal-vue-nuxt/pull/229/files#r327126030